src/components/ExhibitorList/index.js

```diff
@@ -1,781 +1,815 @@
-          {showall ?
-            <div className="showmore-container">
-              <button className="showmorebutton" onClick={() => this.showMore()}>Show All</button>
-            </div> : null}
+            {filteredCompanies.length > 20 && (
+                <div className='showmore-container'>
+                    <button
+                        className='showmorebutton'
+                        onClick={() => setShowMore(true)}
+                    >
+                        Show All
+                    </button>
+                </div>
+            )}
         </div>
-      )
-    } else {
-      return (
-        <div>
-          Exhibitors will be released after summer.
-          </div>
-      )
-    }
-  }
+    )
 }
 
-//TODO: stop using proptypes
-//TODO: reorg of this code into proper places
-
 ExhibitorList.propTypes = {
-  exhibitorName: PropTypes.string,
-  onChangeExhibitorName: PropTypes.func,
-  lastYear: PropTypes.bool
+    exhibitorName: PropTypes.string,
+    onChangeExhibitorName: PropTypes.func,
+    lastYear: PropTypes.bool,
 }
 
-let toExport;
-if (global.window !== undefined) {
-  toExport = addUrlProps({ urlPropsQueryConfig })(ExhibitorList);
-} else {
-  toExport = ExhibitorList;
-}
-export default toExport;
-
-const ExhibitorItem = (props) => {
-  let classname = props.exhibitor.sustainability ? " green" : "";
-  classname += props.exhibitor.diversity ? " purple" : "";
-
-  return (
-    <div id={props.name} role="presentation" className={"exhibitor-box " + classname} onClick={() => props.showModal(props.exhibitor.name)}>
-      <div className="image-container">
-        <img alt='' src={ais + props.exhibitor.logo_squared} />
-      </div>
-      <p> {props.exhibitor.name} </p>
-      {props.exhibitor.location_special === "Diversity Room" ? <div className='corner-special'><img alt='' src='/assets/diversity-black-nolabel.png' /></div> : null}
-      {props.exhibitor.location_special === "Green Room" ? <div className='corner-special'><img alt='' src='/assets/sustainability-black-nolabel.png' /></div> : null}
-    </div>)
+const ExhibitorItem = props => {
+    return (
+        <div
+            id={props.name}
+            role='presentation'
+            className={cls({
+                'exhibitor-box': true,
+                green: props.exhibitor.sustainability,
+                purple: props.exhibitor.diversity,
+            })}
+            onClick={() => props.updateExhibitorName(props.exhibitor.name)}
+        >
+            <div className='image-container'>
+                <img alt='' src={ais + props.exhibitor.logo_squared} />
+            </div>
+            <p> {props.exhibitor.name} </p>
+            {props.exhibitor.location_special === 'Diversity Room' && (
+                <div className='corner-special'>
+                    <img alt='' src='/assets/diversity-black-nolabel.png' />
+                </div>
+            )}
+            {(props.exhibitor.location_special === 'Green Room') &
```

Comment:
```suggestion
            {(props.exhibitor.location_special === 'Green Room') &&
```
Think you missed an extra ampersand which made a 0 display instead of the icon.